ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/validation/ModelConformanceValidator.scala

```diff
@@ -206,9 +206,11 @@ private[transaction] class ModelConformanceValidator(engine: Engine, metrics: Me
       }
 
     val isCausallyMonotonic = transactionEntry.transaction.inputContracts.forall { contractId =>
-      val inputContractState = inputContracts(contractId)
-      val activeAt = Option(inputContractState.getActiveAt).map(parseTimestamp)
-      activeAt.exists(transactionEntry.ledgerEffectiveTime >= _)
+      // We don't check monotonicity for non-existent contracts.
+      inputContracts.get(contractId).forall { inputContractState =>
```

Comment:
Can we improve the naming here? I assume based on the collect that this `inputContracts` represent existing contracts, so it would be nice to capture that in the naming as well.

Reply:
Renamed to `damlContractState`.